client/components/keyed-suggestions/index.jsx

```diff
@@ -172,8 +174,19 @@ class KeyedSuggestions extends Component {
 	onMouseDown = ( event ) => {
 		event.stopPropagation();
 		event.preventDefault();
+
+		const { suggest, recordTracksEvent } = this.props;
 		const suggestion = event.currentTarget.textContent.split( ' ' )[ 0 ];
-		this.props.suggest( suggestion, this.state.isShowTopLevelTerms );
+		if ( this.state.isShowTopLevelTerms ) {
+			// Clean up suggestion that it's passed as, for example, "feature" instead of "feature:".
+			recordTracksEvent( 'search_dropdown_searchby_item_click', {
```

Comment:
Hmm, I'd suggest to rename this to

```
search_dropdown_taxonomy_click
```

and rename the props to

```
taxonomy: feature|subject|etc
```

I'd reserve the term `term` to the actual terms in the query, like `subject:blog`.

What do you think?

Reply:
Sounds good to me 🙂